styles/global.css

```diff
@@ -8,9 +8,33 @@ body,
   border-right: 1px solid #dedede;
 }
 
+.nav-link {
```

Comment:
Mentioned this before, but the importants are an unfortunate side effect of not being able to load our CSS after the react-bootstrap css 🤔 

Will try to figure out a way to fix import ordering but this should be ok for now since it's only the nav